### PR TITLE
[ACTIONS/SOURCES] Megaventory new actions sources

### PR DESCRIPTION
**Pull request by jcortes**
### <samp>🤖 Generated by Copilot at 7b4ca92</samp>

> _We've added some actions and sources_
> _To integrate with Megaventory's courses_
> _They use props and methods_
> _To make API requests_
> _And emit events for new resources_

### CODE REVIEW

File: components/megaventory/megaventory.app.mjs [hunk 1]
```diff
@@ -1,11 +1,195 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+// import utils from "./common/utils.mjs";
```
Comment (vellames) [inline components/megaventory/megaventory.app.mjs:3]: Could you please remove this comment?

File: components/megaventory/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@pipedream/megaventory",
+  "version": "0.0.1",
+  "description": "Pipedream Megaventory Components",
+  "main": "megaventory.app.mjs",
+  "keywords": [
+		"pipedream",
```
Comment (vellames) [inline components/megaventory/package.json:7]: This indentation looks wrong

**Review by vellames — CHANGES_REQUESTED** [review]

**Review by vellames — DISMISSED** [review]
pnpm conflicts :(

**Review by feyzullah — DISMISSED** [review]
LGTM!

**Comment by vunguyenhung** [comment]
Hello everyone, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[Megaventory_3686_1212.pdf](https://github.com/PipedreamHQ/pipedream/files/11258086/Megaventory_3686_1212.pdf)

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Megaventory_3686_1044.pdf](https://github.com/PipedreamHQ/pipedream/files/11268320/Megaventory_3686_1044.pdf)

**Review by feyzullah — DISMISSED** [review]

**Review by feyzullah — APPROVED** [review]

**Pull request merged**